ui/pages/DataManagement/components/PhenotypePri.jsx

```diff
@@ -0,0 +1,31 @@
+import { connect } from 'react-redux'
+
+import { validators } from 'shared/components/form/FormHelpers'
+import { BooleanCheckbox } from 'shared/components/form/Inputs'
+import UploadFormPage from 'shared/components/page/UploadFormPage'
+
+import { getPhenoPriUploadStats } from '../selectors'
+import { uploadPhenoPri } from '../reducers'
+
+const mapStateToProps = state => ({
+  fields: [
```

Comment:
You need to define a `FIELDS` constant, otherwise every time the component gets the chance to rerender it will recreate this list, think it is different, and rerender the whole thing